src/shared/api/services/answerService.ts

```diff
@@ -0,0 +1,56 @@
+import { AxiosResponse } from 'axios';
+
+import httpService from './httpService';
+import { SuccessfulEmptyResponse } from '../types';
+
+type AnswerDto = {
+  activityItemHistoryId: string;
+  answer: Record<string, string>;
+};
+
+type ActivityAnswersRequest = {
+  appletId: string;
+  appletHistoryVersion: string;
+  activityId: string;
+  answers: AnswerDto[];
+};
+
+type ActivityAnswersResponse = SuccessfulEmptyResponse;
+
+type ActivityFlowAnswersRequest = {
+  appletId: string;
+  appletHistoryVersion: string;
+  flowItemHistoryId: string;
```

Comment:
It should be discussed, it is not clear about history properties here

Reply:
Yes, we can talk about it during the next architecture meeting

The endpoint: 
https://api-dev.cmiml.net/docs#/Answers/answer_activity_item_create_answers_activity_items_post